Apache/Ocsinventory/Server/Inventory.pm

```diff
@@ -142,10 +142,6 @@ sub _post_inventory{
   my $result = $Apache::Ocsinventory::CURRENT_CONTEXT{'XML_ENTRY'};
   my $dbh = $Apache::Ocsinventory::CURRENT_CONTEXT{'DBI_HANDLE'};
 
-  set_category();
```

Comment:
Hi @alacn1,

Can you explain why you removed this three lines from Inventory.pm ? Without this lines the fisrt machine's inventory doesn't register sofwtare categories, asset categories and softwares.

Regards,
Charlene